Modix.Services/Utilities/FormatUtilities.cs

```diff
@@ -109,5 +109,19 @@ public static string FormatInfractionCounts(IDictionary<InfractionType, int> cou
 
             return $"This user has {formatted}";
         }
+
+        public static string Sanitize(string text)
+        {
+            var everyoneSanitized = _everyoneMentionRegex.Replace(text, _sanitizedEveryoneMention);
```

Comment:
Could this be just a simple `String.Replace` instead of a regex?

Reply:
Derp. Of course it can. I think I was originally thinking it would be a little fancier, but I ended up trimming it down to the simple replaces without registering that I could replace the regex calls entirely. Fixed.